### PR TITLE
Update sui-breaking-changes.md

### PR DESCRIPTION
**Pull request by randall-Mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: doc/src/doc-updates/sui-breaking-changes.md [hunk 1]
```diff
@@ -50,6 +50,26 @@ New entries added 03/06.
 
 ---
 
+Added 3/9/23
+
+**[Major API breaking changes]** - `GetTransaction` API refactoring
+    - [RPC] `sui_getTransaction` and `sui_multiGetTransaction` now take in an additional optional parameter called `options` that specifies which fields to retrieve (such as  `transaction`, `effects`, `events`, etc). By default, these operations return only the transaction digest.
+    - [TS SDK] Renamed `provider.getTransactionWithEffects` to `provider.getTransactionResponse`. The new method takes in an additional parameter, `SuiTransactionResponseOptions`, to configure which fields to retrieve (such as `transaction`, `effects`, `events`, etc). By default, this method returns only the transaction digest.
```
[the remark this answers is not in the document]
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md:57]: indeed, it looked correct but now it doesn't, thanks for catching

[the remark this answers is not in the document]
Comment (oxade) [reply doc/src/doc-updates/sui-breaking-changes.md]: thanks for addressing swiftly!

[the remark this answers is not in the document]
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md]: fixed, was copied oddly from Notion

**Review by oxade — APPROVED** [review]

File: doc/src/doc-updates/sui-breaking-changes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (randall-Mysten) [reply doc/src/doc-updates/sui-breaking-changes.md]: > thanks for addressing swiftly!

thanks for reviewing!

**Pull request merged**